packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -127,6 +127,28 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
     return t;
   });
 
+  const splitURLPath = useMemo(() => location.pathname.split('/'), []);
```

Comment:
This code looks like it should be in a dedicated helper function. It seems like there are a lot of internals exposes and mashed together all in one function.

Even though we don't have a solution to this yet I think it would be better to just put it in a function. Even if it was in the same directory and not really meant to be a utility.